frontend/src/lib/components/Dropdown/dropdown.tsx

```diff
@@ -14,26 +14,35 @@ import { IconButton } from "../IconButton";
 import { Input } from "../Input";
 import { Virtualization } from "../Virtualization";
 
+export type DropdownOptionGroup<TValue = string> = {
+    label: string;
+    // ? Should groups also be allowed an adornment?
```

Comment:
I think it could be nice to have adornment for the group. Perhaps some dropdowns would like to have adornment only for groups, and not for options.